script/test

```diff
@@ -1,4 +1,7 @@
-#!/bin/bash
+#!/bin/bash -l
+
+source ~/.bashrc
+rvm use default
```

Comment:
Not a huge issue, but locally, because I use rbenv instead and don't have rvm then it triggers a warning about needing to download it. still seems to run the rest of the script fine

Reply:
Ye, its a good point. This test is only used for Jenkins currently, but actually it should be able to execute locally, I've added an if clause to resolve this. There is a difference when running in docker as its not interactive shell. So we need to bring in the context with the source and rvm command.